src/calendar/components/header/header.js

```diff
@@ -0,0 +1,245 @@
+    let minYear = maxYear - 11;
+
+    isPreviousDisable = minDate && minDate.year > minYear;
+    isNextDisable = maxDate && maxDate.year < maxYear;
+  }
+
+  if (disabled) {
+    isPreviousDisable = true;
+    isNextDisable = true;
+  }
+
+  return (
+    <div className="rmdp-header">
+      <div
+        style={{ position: "relative", display: "flex", alignItems: "center" }}
+      >
+        {Array.from(new Set(headerOrder)).map((item, index) => (
+          <Fragment key={index}>{getHeaderItem(item)}</Fragment>
+        ))}
+      </div>
+    </div>
+  );
+
+  function getHeaderItem(item) {
+    switch (item) {
+      case "LEFT_BUTTON":
+        return buttons && getButton("left");
+      case "RIGHT_BUTTON":
+        return buttons && getButton("right");
+      case "MONTH_YEAR":
+      case "YEAR_MONTH":
+        if (fullYear) {
+          return (
+            <div className="rmdp-header-values" style={style}>
+              {!hideYear && date.format("YYYY")}
+            </div>
+          );
+        } else {
+          let items = item
+            .split("_")
+            .filter(
+              (item) =>
+                (item === "MONTH" && !hideMonth) ||
+                (item === "YEAR" && !hideYear)
+            );
+
+          if (items.length > 1) items = [items[0], getSeparator(), items[1]];
+
+          return months.map((month, index) => (
+            <div key={index} className="rmdp-header-values" style={style}>
+              {items.map((item, i) => (
+                <Fragment key={i}>
+                  {getMonthOrYear(item, month, index)}
+                </Fragment>
+              ))}
+            </div>
+          ));
+        }
+      default:
+        return;
+    }
+  }
+
+  function getMonthOrYear(item, month, index) {
+    switch (item) {
+      case "MONTH":
+        return (
+          <span
+            style={{
+              cursor:
+                disabled || disableMonthPicker || onlyMonthPicker
+                  ? "default"
+                  : "pointer",
+            }}
+            onClick={() => !disableMonthPicker && toggle("mustShowMonthPicker")}
+          >
+            {getMonth(month, years[index])}
+          </span>
+        );
+      case "YEAR":
+        return (
+          <span
+            style={{
+              cursor:
+                disabled || disableYearPicker || onlyYearPicker
+                  ? "default"
+                  : "pointer",
+            }}
+            onClick={() => !disableYearPicker && toggle("mustShowYearPicker")}
+          >
+            {getYear(years[index], month)}
+          </span>
+        );
+      default:
+        return item;
+    }
+  }
+
+  function getSeparator() {
+    return !monthYearSeparator ? (
+      isRTL ? (
+        "،"
+      ) : (
+        ","
+      )
+    ) : (
+      <span>{monthYearSeparator}</span>
+    );
+  }
+
+  function getButton(direction) {
+    let handleClick = (e) => {
+        e.preventDefault();
```

Comment:
For some reason this `e.preventDefault()` isn't working, but you could do a check to see if the active element has the correct classes for the arrows before running the month change. Example code:

```
if(document.activeElement.classList.contains('rmdp-arrow-container')) {
  increaseValue(direction === "right" ? 1 : -1);
}
```